pkgs/by-name/sa/saleae-logic-2/package.nix

```diff
@@ -1,14 +1,14 @@
 { lib, fetchurl, makeDesktopItem, appimageTools }:
 let
-  name = "saleae-logic-2";
+  pname = "saleae-logic-2";
   version = "2.4.13";
   src = fetchurl {
     url = "https://downloads.saleae.com/logic2/Logic-${version}-linux-x64.AppImage";
     hash = "sha256-0GIZQKQDY3arDUlxjQKWOHDB3j76xVwkx5H+8q+d0Rc=";
   };
   desktopItem = makeDesktopItem {
-    inherit name;
-    exec = name;
+    name = pname;
+    exec = pname;
```

Comment:
Not blocking, but this feels wrong.

Reply:
Yeah, let's not do this. https://github.com/NixOS/nixpkgs/issues/277994